### PR TITLE
doc(documentation): Tutorial Updates

### PR DESCRIPTION
**Pull request by wodenx**

### CODE REVIEW

**Review by hvanyo — DISMISSED** [review]

File: packages/bodiless-documentation/doc/Development/Guides/IntroToBodilessConcepts.md [hunk 1]
```diff
@@ -562,114 +621,102 @@ system. Usually these tokens would be defined elsewhere (most likely under you
...
-```
-
-Add the following to `Gallery.tsx` just
-after withColoredBorder styling code in previous step:
-
-The FlowContainer takes a design prop that is part of the
-[Design API](../Architecture/FClasses?id=the-design-api). This is how we pass in
-the components that can be used in the grid. we will also use the hoc
-`withTitle` to provide a Title for each component in the selector.
+now we'll use these tokens to define the components which will be available
```
Comment (misterjones) [inline packages/bodiless-documentation/doc/Development/Guides/IntroToBodilessConcepts.md:624]:
```suggestion
Now we'll use these tokens to define the components which will be available
```

File: packages/bodiless-documentation/doc/Development/Guides/IntroToBodilessConcepts.md [hunk 2]
```diff
@@ -562,114 +621,102 @@ system. Usually these tokens would be defined elsewhere (most likely under you
...
-  ```
+The `design` object here is similar to the one we used above to populate the
+rich text editor.  It is a keyed set of HOC's which are used to produce
+a set of available components.  In both cases. we start with the special
+`replaceWith` HOC, which injects the specified component as a starting point.
 
-- Then remove the children where the `<Body />` tag appears - change
+Next, replace the `Body` of the gallery with a flow container (a flexbox-based
+layout builder) which makes our three tiles available for placement by a content
+edtior.
```
Comment (misterjones) [inline packages/bodiless-documentation/doc/Development/Guides/IntroToBodilessConcepts.md:665]:
```suggestion
editor.
```

**Review by misterjones — APPROVED** [review]
Reviewed markdown files. ✅ 

Provided suggestions for spelling and minor formatting corrections.

**Review by hvanyo — APPROVED** [review]

File: packages/bodiless-documentation/doc/Development/Guides/DesignElementConcepts.md [hunk 3]
```diff
@@ -23,36 +23,170 @@ to apply the same design system to multiple sites, extending it as needed.
...
-   with `import { withEditorBasic } from '../../../components/Editors';`
-1. Replace `const Body = withSimpleEditor('body', 'Body')(Fragment);`
-   with `const Body = withEditorBasic('body', 'Body')(Fragment);`
+custom editors on individual pages.  Similarly, a sites typography should also be
+consistent.
+
+### Move typography tokens.
+
+If it does not exist already, create a `/src/components/Elements` directory
+at the root of your project.  In this, create a `tokens.ts` file, and ddd
```
Comment (misterjones) [inline packages/bodiless-documentation/doc/Development/Guides/DesignElementConcepts.md:42]:
```suggestion
at the root of your project.  In this, create a `tokens.ts` file, and add
```

File: packages/bodiless-documentation/doc/Development/Guides/DesignElementConcepts.md [hunk 4]
```diff
@@ -23,36 +23,170 @@ to apply the same design system to multiple sites, extending it as needed.
...
+utility from Lodash: it accepts a list of higher order components and returns a
+new higher order component which composes them. However, `asToken` adds some
+additional functionality. In this example, we use it to specify
+*metadata* which should attached to the resulting token.
+
+This metadata is useful for several reasons:
+- It encourages us to think of tokens in a structured way.
+- It provides a basis for browsing and editing tokens using the Bodiless token
+  browser (experimental).
+- It allows *fitering* of tokens (we'll get to this in a later tutorial).
```
Comment (misterjones) [inline packages/bodiless-documentation/doc/Development/Guides/DesignElementConcepts.md:103]:
```suggestion
- It allows *filtering* of tokens (we'll get to this in a later tutorial).
```

**Review by misterjones — APPROVED** [review]
Re-reviewed markdown.

Couple quick fixes.

File: packages/fclasses/src/Design.tsx [hunk 5]
```diff
@@ -98,6 +98,36 @@ export const replaceable = <P extends object> (Component:ComponentOrTag<P>): Com
...
+ * HOC which removes the original component and renders the replacement instead.
+ *
+ * @see `replaceWith`
+ * @example
+ *  ```js
+ *  const ExampleBase = ({ components: C, ...rest }) => <C.Tag {...rest} />;
+ *  const ExampleClean = designable({ Tag: Span })(ExampleBase);
+ *  const Example = withDesign({
+ *    Tag: addClasses('text-blue'),
+ *  })(Example); // <span className="text-blue" />
```
Comment (dewen) [inline packages/fclasses/src/Design.tsx:122]: Example => ExampleClean

File: packages/fclasses/src/Design.tsx [hunk 6]
```diff
@@ -195,16 +225,23 @@ export const withDesign = <C extends DesignableComponents>(
...
  * @example
- * ```
- * const Start = () => <div />
- * const Replaced = replaceWith('span')(Start);
- * <Start /> === '<div />'
- * <Replaced /> ==== '<span />'
+ * ```js
+ * import { Div, replaceWith } from `@bodiless/fclasses`;
+ * const StartBase = Div; // <div />
+ * const Start = addClasses('text-blue')(Start); // <div className="text-blue" />
```
Comment (dewen) [inline packages/fclasses/src/Design.tsx:243]: (Start) => (StartBase)

File: packages/bodiless-components/doc/Link.md [hunk 7]
```diff
@@ -60,25 +60,34 @@ Toggle" /></LinkToggle>
...
+## Customizing link behavior
+
+`asBodilessLink`, like any other `asBodiless...` function, accepts a node key, a
+default value and a `useOverrides` hook, and returns an HOC which makes a link
+editable.  You can use the `useOverrides` hook to customize the link editor. In
+particular, you can define a custom function which will be used to normalize
+the href when the link is saved or displayed. By default, `asBodilessLink`
+performs some basic normalization.  Here's an example of how to disable it:
+
+```ts
```
Comment (dewen) [inline packages/bodiless-components/doc/Link.md:82]: adding import A to be more clear below
`import { A } from `@bodiless/fclasses`;`

**Review by misterjones — APPROVED** [review]
Re-reviewed markdown.

Added suggestions for minor tweaks.

**Pull request merged**